### PR TITLE
Add 11.10 release notes

### PR DESCRIPTION
**Pull request by tas50**
#### Description

New week!

#### Types of changes

- [ ] Functional documentation bug fix (i.e., broken link or some other busted behavior)
- [ ] New functional doc capabilities (i.e., filter search results)
- [x] New content
- [ ] Revision to existing content
- [ ] Chore (non-breaking change that does not add functionality or fix an issue)

#### Checklist

- [x] I have read the **README** document about contributing to this repo.
- [x] I have tested my changes locally and there are no issues.
- [x] All commits are signed.

### CODE REVIEW

File: docs/mql/resources/arista-pack/arista.eos.md [hunk 1]
```diff
@@ -35,30 +35,30 @@ Arista EOS resource
 
 Arista EOS Version
 
-```coffeescript
```
Comment (tas50) [inline docs/mql/resources/arista-pack/arista.eos.md:38]: These are intentionally coming back due to how we generate these files. It'll get fixed one way or the other at a later date.

File: docs/mql/resources/os-pack/windows.optionalfeature.md [hunk 2]
```diff
@@ -25,17 +25,3 @@ windows.optionalFeature(name string)
 | description | string | Feature description            |
 | enabled     | bool   | Whether the feature is enabled |
 | state       | int    | Feature installation state     |
-
-**Examples**
```
Comment (tas50) [inline docs/mql/resources/os-pack/windows.optionalfeature.md:29]: Intentionally removed for now

File: releases/2024-06-25-mondoo-11.10-is-out.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+### Secure Microsoft Exchange Server 2019
+
+https://github.com/mondoohq/cnspec-enterprise-policies/pull/917
+
+### Continuous IaC discovery in GitHub and GitLab
+
+https://github.com/mondoohq/console/pull/4550
+
+### Continuos GitHub Enterprise scanning
```
Comment (misterpantz) [inline releases/2024-06-25-mondoo-11.10-is-out.md:28]: "Continuos" is a baroque musical term. Is our spellcheck seriously allowing it? 😆

File: releases/2024-06-25-mondoo-11.10-is-out.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
+---
+slug: mondoo-11.10-is-out/
+title: Mondoo 11.10 is out!
+description: Announcing the 11.10 release of Mondoo, with CIS benchmarks for Exchange Server 2019, faster Azure scans, new resouces, and more!
```
Comment (misterpantz) [inline releases/2024-06-25-mondoo-11.10-is-out.md]:
```suggestion
description: Announcing the 11.10 release of Mondoo, with CIS benchmarks for Exchange Server 2019, faster Azure scans, new resources, and more!
```

File: releases/2024-06-25-mondoo-11.10-is-out.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
+---
+slug: mondoo-11.10-is-out/
+title: Mondoo 11.10 is out!
+description: Announcing the 11.10 release of Mondoo, with CIS benchmarks for Exchange Server 2019, faster Azure scans, new resources, and more!
```
Comment (misterpantz) [inline releases/2024-06-25-mondoo-11.10-is-out.md:4]:
```suggestion
description: Announcing the 11.10 release of Mondoo, with Windows filesystem scanning, expanded IaC reporting, and more!
```

File: releases/2024-06-25-mondoo-11.10-is-out.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+### Secure Microsoft Exchange Server 2019
+
+Validate your Microsoft Exchange Server 2019 instance against the latest CIS benchmarks with our new policy, which contains 52 new tests to ensure that you follow best practices and security recommendations.
+
+![Exchange Server 2019 Checks](/img/releases/2024-06-25-mondoo-11.10-is-out/exchange.png)
+
+## 🧹 IMPROVEMENTS
+
+### Improved CVE and advisories affected assets lists
```
Comment (misterpantz) [inline releases/2024-06-25-mondoo-11.10-is-out.md:28]:
```suggestion
### Improved affected assets lists for CVE and advisories
```

File: releases/2024-06-25-mondoo-11.10-is-out.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+- Automatically enable the generated compliance evidence policy for custom compliance frameworks.
+- Render check and data query markdown in compliance reports.
+- Add the number of queries to the compliance report summary page.
+- Fetch the latest VMware advisories that are now published by Broadcom.
+- Improve the output of queries in Kubernetes policies.
+- Improve reliability of the "Ensure 'Debug programs' is set to 'Administrators'" check in Windows policies.
+- Improve connection error output for Google Cloud, Azure, and AWS resources.
+- Improve filters on the Azure query pack to prevent failures running queries.
+- Fix missing check counts in the asset check overview.
+- Improve reliability of the CIS Amazon EKS benchmark "Ensure Network Policy is Enabled and set as appropriate" and "Ensure clusters are created with Private Nodes" checks.
```
Comment (misterpantz) [inline releases/2024-06-25-mondoo-11.10-is-out.md:77]: Well this capitalization is wonky, but I suspect that's how the check names are capitalized, so meh.

File: releases/2024-06-25-mondoo-11.10-is-out.md [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+- Expand the NSA PowerShell policy's "Disable and uninstall the deprecated PowerShell v2" check to work on both desktop and server Windows installations.
+- Improve reliability of AWS IAM credential report fetching.
```
Comment (misterpantz) [inline releases/2024-06-25-mondoo-11.10-is-out.md:79]:
```suggestion
- More reliably fetch AWS IAM credential reports.
```

File: releases/2024-06-25-mondoo-11.10-is-out.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+- Improve reliability of Azure managed identity credentials fetching.
```
Comment (misterpantz) [inline releases/2024-06-25-mondoo-11.10-is-out.md:80]:
```suggestion
- More reliably fetch Azure managed identity credentials.
```

**Review by misterpantz — COMMENTED** [review]
Looks great! Just a couple suggestions.

File: releases/2024-06-25-mondoo-11.10-is-out.md [hunk 10]
```diff
@@ -0,0 +1,81 @@
...
+title: Mondoo 11.10 is out!
+description: Announcing the 11.10 release of Mondoo, with CIS benchmarks for Exchange Server 2019, faster Azure scans, new resources, and more!
+author: Tim Smith
+author_title: Mondoo Core Team
+author_url: https://github.com/tas50
+image: /img/featured_img/release-feature.jpg
+tags: [release, mondoo]
+---
+
+## 🥳 Mondoo 11.10 is out! This release includes CIS benchmarks for Exchange Server 2019, faster Azure scans, new resources, and more!
```
Comment (misterpantz) [inline releases/2024-06-25-mondoo-11.10-is-out.md:12]: Let's accurately summarize this now that we know what is in it. :) 

```suggestion
## 🥳 Mondoo 11.10 is out! This release includes CIS benchmarks for Exchange Server 2019, improvements to the CVE and advisory UI, new resources, and more!
```

File: releases/2024-06-25-mondoo-11.10-is-out.md (hunk 5 above)
Comment (misterpantz) [reply releases/2024-06-25-mondoo-11.10-is-out.md:4]: Okay to make this accurate, how about:
```suggestion
description: Announcing the 11.10 release of Mondoo, with CIS benchmarks for Exchange Server 2019, improvements to the CVE and advisory UI, new resources, and more!
```

**Review by misterpantz — APPROVED** [review]

**Pull request merged**